cmd/root.go

```diff
@@ -167,6 +169,11 @@ func setRootFlags(flags *pflag.FlagSet) {
 		"ignore matches for vulnerabilities that are fixed",
 	)
 
+	flags.BoolP(
+		"by-cve", "", false,
```

Comment:
Yea I can see how a user could conflate it. I also agree none of the current options standout as a clear winner. I like that --by-cve does not show GHSA. That feels right. The inverse --by-ghsa would not show CVE.

I also ran it locally and it feels right for now, but might be biased since it was the first one I read.

I say ship, and if er end up hating it we can change up the ergonomics if it still feels bad.

Keith also had a good point of eventually making this the default.